doc/tutorials/acrn_on_qemu.rst

```diff
@@ -186,10 +193,14 @@ Install ACRN Hypervisor
          insmod ext2
 
          echo 'Loading ACRN hypervisor ...'
-         multiboot --quirk-modules-after-kernel /boot/acrn.32.out
-         module /boot/bzImage Linux_bzImage
+         multiboot --quirk-modules-after-kernel /boot/acrn.32.out  root=/dev/vda5
+         module /boot/vmlinuz-5.10.115-acrn-service-vm Linux_bzImage
       }
 
+     .. note::
+     If your rootfs partition isn't vda5, please change it to match with yours.
```

Comment:
I believe this paragraph is misaligned (haven't checked the rendering though). @fuzhongl can you double-check that this appears as it should?

@dbkinder , can you also suggest some phrasing for this note?